.github/workflows/ci.yml

```
@@ -70,6 +70,28 @@ jobs:
           command: check
           args: --features=stm32f303xc,stm32-usbd,rt,can,ld --lib --examples
 
+  check-min-deps:
+    name: Check Minimal Dependencies
```

```suggestion
    name: Check Minimal Dependcy Versions
```

Maybe this is more descriptive, because of the usage of [`-Z minimal-versions`](https://doc.rust-lang.org/cargo/reference/unstable.html?highlight=-Z%20min#minimal-versions)?